temba/contacts/templatetags/contacts.py

```diff
@@ -199,7 +199,7 @@ def history_class(item):
     if isinstance(item, Msg):
         if item.media and item.media[:6] == 'video:':
             css = '%s %s' % (css, 'video')
-        if item.direction or item.recipient_count:
+        if item.direction or (item.broadcast and item.broadcast.recipient_count > 1):
```

Comment:
wrt the prefetching, this stuff is called by the Contact's History SmartReadView which does `contact.get_activity()` which fetches the [`broadcast` using `select_related`](https://github.com/rapidpro/rapidpro/blob/406dc784bb40b799021bcd41386be37629a46ad3/temba/contacts/models.py#L526).